docs/schema/2.0/manifest.schema.json

```diff
@@ -125,7 +125,7 @@
         },
         "version": {
           "type": "string",
-          "pattern": "^[0-9]+\\.[0-9]+$",
+          "pattern": "^[0-9]+(\\.[0-9]+)?$",
```

Comment:
Do we need that major.minor format? Or shall we just change the type to number?

Reply:
I would not close the opportunity yet. It won't be user input anyway, we can still keep both options for us.